hack/windows/Clean-AntreaNetwork.ps1

```diff
@@ -15,33 +15,122 @@ function GetHnsnetworkId($NetName) {
     }
     return $null
 }
-$AntreaHnsNetworkName = "antrea-hnsnetwork"
 
-Write-Host "Delete OVS bridge: br-int"
-ovs-vsctl.exe --no-wait --if-exists del-br br-int
-$MaxRetryCount = 10
-$RetryCountRange = 1..$MaxRetryCount
-$BrIntDeleted = $false
-foreach ($RetryCount in $RetryCountRange) {
-    Write-Host "Waiting for OVS bridge deletion complete ($RetryCount/$MaxRetryCount)..."
-    $BrIntAdapter = $(Get-NetAdapter br-int -ErrorAction SilentlyContinue)
-    if ($BrIntAdapter -eq $null) {
-        $BrIntDeleted = $true
-        break
+function ClearHyperVBinding($adapter) {
+    $status= $(Get-NetAdapterBinding -Name $adapter.Name -ComponentID vms_pp).Enabled
+    if ($status -EQ "False") {
+        Set-NetAdapterBinding -Name $adapter.Name -ComponentID vms_pp -Enabled $False
+    }
+}
+
+function ClearHyperVBindingOnAdapter($adapterName) {
+    if ($adapterName -NE "") {
+        $adapter = $(Get-NetAdapter -Name $adapterName)
+        if ($adapter -eq $null) {
+            return
+        }
+        ClearHyperVBinding($adapter)
+    } else {
+        $adapters= $(Get-NetAdapter | ? Virtual -EQ $false)
+        if ($adapters -eq $null) {
+            Write-Host "Physical network adapters not found"
+            return
+        }
+        foreach ($adapter in $adapters) {
+            ClearHyperVBinding($adapter)
+        }
+    }
+}
+
+function ResetOVSService() {
+    $ovsVswitchdSvc = Get-Service ovs-vswitchd -ErrorAction Ignore
+    if ($ovsVswitchdSvc -EQ $null) {
+        return
+    }
+    $ovsStatus = $(Get-Service ovs-vswitchd).Status
+    if ("$ovsStatus" -EQ "StartPending") {
+        sc.exe delete ovs-vswitchd
+        stop-service ovsdb-server
+
+        # $OVSInstallDir is where OVS is installed. It is the path argument when using Install-OVS.ps1.
+        $OVSInstallDir="C:\openvswitch"
```

Comment:
Better to define these paths at the beginning of the file, so users can find them easier?

Reply:
Sure, updated.